cinema_service/settings.py

```diff
@@ -20,12 +22,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv("DEBUG") != "False"
```

Comment:
The DEBUG setting is being loaded from an environment variable, which is great. However, it's being compared to the string "False". This could lead to unexpected behavior if the environment variable is not set or if it's set to a value other than "False". It would be better to compare it to the string "True" to avoid these potential problems.